js/targeted-content.js

```diff
@@ -3,7 +3,8 @@ const initTargetedContent = () => {
         const content = document.getElementsByClassName(
             'cads-targeted-content'
         );
-        content.forEach(item => {
+        for (let i = 0; i < content.length; i++) {
```

Comment:
I think we should check if content is not null/undefined here and has a length before running the for loop to avoid weird stuff.

Reply:
That's why I have it in a try block.